swagger.yaml

```
@@ -56,24 +67,52 @@ paths:
         200:
           description: OK
           schema:
-            $ref: "#/definitions/SearchResponse"
+            $ref: "#/definitions/DepartmentResponse"
         400:
           description: Query term not specified
         500:
           description: Internal server error
 
-  /departments/search:
+  /health:
     get:
       tags:
-        - department
-      summary: "Department query API"
-      description: "Department query API which returns matching links to other departments"
+        - health
+      summary: "Health check API"
+      description: "API to query the app for current health status"
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/Health"
+        500:
+          description: Internal Server Error
+
+  /search:
+    get:
+      tags:
+        - search
+      summary: "Default ONS query API"
+      description: "Default ONS query API which combines the content with type counts"
       parameters:
         - in: query
           name: q
-          description: "Query search term"
+          description: "Query search term."
           type: string
           required: true
+        - in: query
+          name: content_type
+          description: "Comma separated list of content types to be returned."
+          type: array
+          uniqueItems: true
+          items:
+            type: string
+          collectionFormat: csv
+          required: false
+        - in: query
+          name: sort
+          description: "The order to return the results."
+          type: string
+          required: false
```

This might need a list of available sorting options, and a default if there is one. For example:
```suggestion
        - in: query
          name: sort
          description: "The order to return the results."
          type: string
          enum:
              - publish_date
              - name
          default: name
          required: false
```

---

I think that's out of the scope of this ticket. I can raise this with Nathan though.

---

See comment above.